src/models/Eatery.ts

```diff
@@ -0,0 +1,18 @@
+export class Eatery {
+  // name: string
```

Comment:
remove comment

Reply:
Left this there because for the MVP we are only using DummyData and the `name` of the eatery is not very useful. The `name` will be useful when connect with API request. As such, left as a reminder for further development efforts